app/controllers/events_controller.rb

```diff
@@ -23,15 +23,15 @@ def index
   end
 
   def show
-    event = Event.find_by_slug(params[:id])
+    event = Event.find_by(slug: params[:id])
 
     @event = EventPresenter.new(event)
     @host_address = AddressPresenter.new(@event.venue.address)
 
     if logged_in?
       invitation = Invitation.find_by(member: current_user, event: event, attending: true)
       if invitation
-        redirect_to event_invitation_path(@event, invitation) and return
+        redirect_to event_invitation_path(@event, invitation) && return
```

Comment:
I think this should be

```ruby
return redirect_to event_invitation_path(@event, invitation)
```

Can you please double check if there is a spec for that?